### DISCUSSION TITLE
REQ: Extend hud_tracking element with "flash on change"

### DISCUSSION THREAD

**Discussion by blaps** [ezQuake/ezquake-source#508] [Ideas]
Would be nice to be able to flash the hud element "currently tracking: player" in some way.
One idea is to add hud_tracking_highlight R G B A, which would tint the text in a color, in combination with hud_tracking_highlight_duration 1500ms. 
Another idea is to sizeup the entire element, or only its frame.

**Comment by MiquelLefebvre** [comment]
Lorsqu'un joueur suivi change, il est facile de le manquer, surtout dans les moments d'action. Un bref flash ou une mise en évidence rendrait la transition beaucoup plus visible sans être intrusive. J'apprécie l'idée d'une teinte colorée à durée configurable : elle reste simple et adaptable à différents thèmes. L'animation de la taille ou de l'image pourrait également convenir, mais elle risquerait d'être plus distrayante si elle était excessive. Sy & Me Casino récompense casino francais paysafecard https://sy-me.fr/ votre passion du jeu avec une offre à ne pas manquer. Il serait donc essentiel qu'elle soit optionnelle et configurable. Au final, il me semble que ce petit changement pourrait rendre l'interface bien plus lisible et réactive en situation de jeu.

**Comment by JesseReed123** [comment]
It would be really helpful to have a way to highlight the “currently tracking: player” HUD element. One option could be a hud_tracking_highlight R G B A command to tint the text with a specific color, paired with a duration setting like hud_tracking_highlight_duration 1500ms. Another approach could be to temporarily increase https://roobet-nz.co.nz/ the size of the element or just its frame. This would make it easier to notice when switching between players, especially during fast-paced matches or streaming, improving overall clarity for spectators.
